deploy/utils.sh

```diff
@@ -157,6 +154,117 @@ function deploy_env {
 
   export CONJUR_APPLIANCE_URL=$conjur_appliance_url
   export CONJUR_AUTHN_URL=$conjur_authenticator_url
+}
+
+fetch_ssl() {
+  selector="role=follower"
+  cert_location="/opt/conjur/etc/ssl/conjur.pem"
+  if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+    selector="app=conjur-cli"
+    export cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
+  fi
+
+  export conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
+}
+
+setup_helm_environment() {
+  set_namespace $CONJUR_NAMESPACE_NAME
+
+  # Configure authn and login Conjur urls
+  configure_env_url
+
+  # Fetch SSL cert from Conjur/DAP
+  fetch_ssl
```

Comment:
this indicates that the method name is not explicit enough. you can change it to `fetch_ssl_from_conjur` and remove the comment

Reply:
you are right! updating